libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -2193,7 +2197,12 @@ void WaveTrack::Paste(double t0, const Track &src)
 {
    assert(IsLeader()); // pre of Track::Paste
    if (const auto other = dynamic_cast<const WaveTrack*>(&src))
-      PasteWaveTrack(t0, *other);
+   {
+      // Currently `Paste` isn't used by code that wants the newer "don't merge
+      // when copy/pasting" behaviour ...
+      constexpr auto merge = false;
+      PasteWaveTrack(t0, *other, merge);
```

Comment:
I'm confused.  `true` not `false` must be passed if the intention is NOT to change behavior in this path, as the comment suggests.  That is, to pass the default for `split` to `PasteOne`, which is `true`, but is also the negation of `merge`.

Places where you can reach this: 
- Repeat effect
- Paulstretch
- Cut, copy, split cut by label
- Track::SyncLockAdjust (but is this in fact unreachable because of overriding functions?)

Other places, but where I think the destination track is always empty so it does not matter:
- Stereo to mono
- Benchmark

Reply:
Right, should be `true`.
Your mention of Paulstretch made me uncover another bug, where when applied to a stretched clip results in the not-enough-room error. I'll have a fix commit for that.